src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -39,6 +40,7 @@
 #include <libnl3/netlink/genl/ctrl.h>
 #include <libnl3/netlink/genl/mngt.h>
 
+#include <experimental/filesystem>
```

Comment:
Nit: we probably need to do the same awkward `__has_include` hack that lots of other files are doing in Ceph ... or create a "include/ceph_filesystem.h" helper that just does it once and pulls the filesystem namespace into "ceph::fs" (or similar)

Reply:
This comment is still unaddressed

Reply:
@dillaman  sorry for this. It should be addressed now.